### DISCUSSION TITLE
i18n for open-graph images using app router and dynamic [lang] dir

### DISCUSSION THREAD

**Discussion by maxibenner** [vercel/next.js#55502] [App Router]
I just finished i18n for my site. The only thing missing are the appropriate open-graph images for each language. However, I haven't been able to figure out how to do it. So far I tried 1) accessing the appropriate images from the public folder using the Metadata object and 2) exporting a Next Image component from the opengraph-image.tsx file. I wasn't able to get either to work. The documentation doesn't seem clear to me and I am struggling with understanding the required paths and principles behind this.

Any help would be greatly appreciated.

**Comment by silas-v1** [comment]
What errors do you see?

**Reply by maxibenner** [reply]
Thank you for checking in. The error I got was the open-graph images not showing up in the og debugger or when sharing.

I did manage to solve the problem by using the dynamic .tsx open-graph file and using an svg inside of it instead of an image.
